### ISSUE TITLE
Please update npm package local variables is not working 

### ISSUE THREAD

**Issue by Imsatis** [SelectTransform/st.js#37]
**This example is not working with nodejs package..**

10. Local Variables //https://selecttransform.github.io/site/transform.html
You can use #let API to declare local variables. The #let API takes an array as a paremeter, which has two elements:

1.The first parameter: the {{#let}} statement which assigns any value to a variable
2.The second parameter: the actual expression that will be evaluated

**Comment by jmiddleton** [comment]
Yes please, could you create a new npm package, that would be awesome...

**Comment by leogps** [comment]
Can the new npm package be created? Any update on this please?
